README.md

```diff
@@ -32,12 +32,12 @@ const unregister = fetchIntercept.register({
         return Promise.reject(error);
     },
 
-    response: function (response) {
+    response: function (response, request) {
```

Comment:
You can access the `request` inside the `response` object. The same applies to the `error` on the `responseError` interceptor.

Reply:
But the body of request is already used we wouldn't be able to re-use it for any retry mechanism it would be better if we clone it and send it back so that dev's can extend it for further usage